commands/repl.js

```diff
@@ -5,10 +5,16 @@ module.exports = {
     handler: async (argv) => {
         const repl = require('repl');
         const context = repl.start('> ').context;
-        context.nearlib = require('near-api-js');
+        context.nearAPI = require('near-api-js');
+        context.nearlib = new Proxy(context.nearAPI, {
```

Comment:
let's just remove it, pretty sure nobody really depends on this yet (`near repl` has been broken for a while before anyone noticed)